data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,848 @@
+		"hit force" 40
+	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. The Acuit Artillery Cannon outranges most weapons in Gegno space, and has a rate of fire higher than most other weapons, making it a very efficient bombardment weapon from afar.`
+
+effect "acuit hit"
+	sprite "effect/acuit hit"
+		"frame rate" 18
+		"no repeat"
+	"lifetime" 18
+	"velocity scale" -0.004
+
+
+outfit "Guile Pulse Laser"
+	category "Guns"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 109000
+	"thumbnail" "outfit/guile pulse laser"
+	"mass" 15
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	weapon
+		"sprite" "projectile/guile pulse laser"
+			"frame rate" 6
+		sound "guile pulse"
+		"hit effect" "heavy laser impact"
+		"inaccuracy" 0.2
+		"velocity" 150
+		"lifetime" 1
+		"reload" 6
+		"burst count" 3
+		"burst reload" 1
+		"firing energy" 3.7
+		"firing heat" 10
+		"shield damage" 12.4
+		"hull damage" 20.2
+		"cluster"
+	description `The Guile Pulse Laser is a recent experimental advancement in Scin plasma weapon technology. Rather than rapid firing plasma slugs, this weapon fires concentrated pulses of energy that only last for a brief moment. Each blast impacts the target ship almost instantaneously with a powerful hit, and a consistent amount of fire from Guile Lasers is extremely effective against slow moving targets.` 
+	description `	Guile Pulse Lasers have noticeable drawbacks as they quickly consume energy and generate notable heat per shot. Unlike a beam-type weapon, they are incapable of consistent output, and have a fairly short range.`
+
+
+outfit "Excavator"
+	category "Guns"
+	cost 0
+	"thumbnail" "outfit/mouthparts"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
+	"gun ports" -1
+	weapon
+		sound "drill"
+		"hit effect" "drilling"
+		"inaccuracy" 90
+		"velocity" 18
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 4
+		"firing heat" 100
+		"shield damage" 8
+		"hull damage" 14
+		"minable damage" 104
+		"hit force" -4
+	description `Excavator Drills are large drills designed by the Gegno that rip materials out of asteroids in an efficient manner. These types of drill systems are only found on ships specifically designed around them as they require substantial existing infrastructure to be functional.`
+
+effect "drilling"
+	sprite "effect/smoke"
+		"no repeat"
+	sound "explosion tiny"
+	"lifetime" 20
+	"random angle" 360
+	"random frame rate" 5
+	"random spin" 2
+	"random velocity" 0.4
+	"velocity scale" -0.01
+
+
+outfit "Savagery Bludgeon"
+	category "Guns"
+	cost 0
+	"thumbnail" "outfit/unknown"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"gun ports" -1
+	"unplunderable" 1
+	weapon
+		"hit effect" "bludgeon impact"
+		"inaccuracy" 360
+		"velocity" 30
+		"lifetime" 1
+		"reload" 1
+		"shield damage" 8
+		"hull damage" 40
+		"hit force" 140
```

Comment:
I still think this should have a longer reload.

Reply:
Is this a mace attached to a swing arm so it can be swung like a wrecking ball at an opponent? Or is this like the giant ramming protrusion from the from of a triream or other naval ship?

If it is a mace on a swing arm, then a reasonably fast reload seems appropriate.
If it is a ramming protrusion on the front, then I'd say the hull damage should be upped a bit but the reload made vastly longer. Long enough to need ships to disengage, turn, and take another run at their target for it to "reload."